tests/unit/packet/PublishTest.php

```diff
@@ -225,65 +244,67 @@ public function testParseWithQos($qos, $bit)
     {
         $input =
             chr($bit) .
-            chr(4) .
+//            chr(4) .
```

Comment:
I don't know why the change I'm proposing expects there to not be the Line Feed character `char(10)` on the end.